.eslintrc.js

```diff
@@ -68,5 +68,7 @@ module.exports = {
     'sonarjs/cognitive-complexity': 'warn',
     '@typescript-eslint/no-misused-promises': 'warn',
     '@typescript-eslint/ban-ts-comment': 'warn',
+    // many times you dont want to add everything to deps to avoid unnecessary triggers of hooks. for example useEffect.
+    'react-hooks/exhaustive-deps': 'off',
```

Comment:
We definitely want this rule enabled. If needed, you can always disabled in locally.